metrics.rst

```diff
@@ -7,8 +9,44 @@ Netdata is an open-source, real-time, performance monitoring and troubleshooting
 It provides comprehensive insights into the performance and health of systems and applications through visualizations and detailed metrics.
 Netdata is designed to be lightweight, fast, and easy to use.
 
-The Netdata web user interface is accessible at the LAN IP address of the firewall on port ``19999``, with the HTTP protocol.
-Address example: ``http://192.168.1.1:19999``
+Netdata is enabled by default on NethSecurity and it is accessible from the LAN network. To access it, go to the ``Report`` page
+and click :guilabel:`Open report` button from the ``Real time report`` tab.
 
 Netdata metrics are saved in RAM and will be reset at very machine reboot.
 If the firewall is connected to the :ref:`remote controller <controller-section>`, metrics will be stored to the controller itself and preserved across reboots.
+
+.. _alert-section:
+
+Alerts
+======
+
+The alert system leverages the power of the Netdata engine for efficient monitoring and alerting.
+
+The alert system prioritizes only those alerts that have the potential to disrupt or compromise the firewall's functionality.
+By focusing on critical indicators, administrators can efficiently address issues that pose a genuine threat to the security and operation of the firewall.
+
+If the server has a valid :ref:`subscription-section`, alert notifications are seamlessly sent to remote servers for centralized monitoring and management.
+Both ``my.nethesis.it`` and ``my.nethserver.com`` serve as central hubs for receiving alerts, allowing administrators to stay informed about the firewall's
```

Comment:
Both ``my.nethesis.it`` and ``my.nethserver.com`` serve as central hubs for receiving alerts, allowing administrators to stay informed about the firewall's status and promptly respond to any critical situations.